src/App/Application.h

```diff
@@ -120,6 +120,33 @@ class AppExport Application
     App::Document* getActiveDocument(void) const;
     /// Retrieve a named document
     App::Document* getDocument(const char *Name) const;
+
+    /// Path matching mode for getDocumentByPath()
+    enum PathMatchMode {

```

Comment:
This can be an enum class (see http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-class).